### PR TITLE
Fixing Map loading

### PR DESCRIPTION
**Pull request by alaxalves**
This PR is a little modified version of #882 to work with Rails 5.

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@sashadev-sky commenting myself here so i get updates on this

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +15,12 @@ cache:
    - $PWD/public/lib/
    - $PWD/vendor/.bundle/
 
+before_install:
+  - yarn add mkdirp graceful-fs
+  - npm i -g npm@6.9.0
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.17.3
+  - export PATH="$HOME/.yarn/bin:$PATH"
+
```
Comment (kaustubh-nair) [inline .travis.yml]: What is this supposed to be doing?

Comment (sashadev-sky) [reply .travis.yml]: I keep this as my yarn export path: 
```JS
export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
```

Comment (alaxalves) [reply .travis.yml]: @kaustubh-nair https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn

**Comment by alaxalves** [comment]
image
Experiencing the super annoyin issue rn, but I think it's because of our dependencies folder definition in `.yarnrc` file. Yarn looks for the dependencies by default on `node_modules/` folder and not the `public/lib/` defined by us.

**Comment by alaxalves** [comment]
> image
> Experiencing the super annoyin issue rn, but I think it's because of our dependencies folder definition in `.yarnrc` file. Yarn looks for the dependencies by default on `node_modules/` folder and not the `public/lib/` defined by us.

Just confirmed this at: https://github.com/sass/node-sass/issues/2050#issuecomment-317233552 and https://github.com/sass/node-sass/issues/2050#issuecomment-317212071.

So basically, we should use `node_modules/`, since we don't need any other patterns, e.g. for production, I think it's a good idea to keep the NPM standard.

**Comment by jywarren** [comment]
> Experiencing this super annoyin issue rn, but I think it's because of our dependencies folder definition in .yarnrc file. Yarn looks for the dependencies by default on node_modules/ folder and not the public/lib/ defined by us.

Hi folks, this change to `.yarnrc` may be causing us some trouble... the `.yarnrc` setting is needed to preserve public accessibility of yarn assets such as images, stylesheets, fonts. 

I added it back in in `main` branch but we should try to work this problem. I'll copy this all to a new issue for discussion, thanks!

**Pull request merged**